### PR TITLE
Fixes iron duplication bug

### LINKED ISSUE #76191: Turning Iron into Floor Tiles doubles their value
## Round ID:
Not really relevant you can duplicate this issue rather quickly

## Testmerges:
No

## Reproduction:
Remove 1 iron sheet from a department lathe, notice the ore silo value decrease by 100
Click on the iron sheet in your hand to craft 4x floor tile (NOT material floor tile)
Place the crafting result back into the lathe for a material total of 200

### PR DESCRIPTION
**Pull request by Peliex**
## About The Pull Request

Each regular floor tile was worth 50 before, fixed to be 25 instead
## Why It's Good For The Game

fixes #76191
Prevents turning 5000 iron into 10000 iron in less than 10 seconds
## Changelog
:cl:
fix: changed station floor tiles to be worth 25 instead of 50, one quarter of an iron sheet.
/:cl:

### CODE REVIEW

**Comment by Peliex** [comment]
#76191

**Comment by Sealed101** [comment]
put `fixes #76191` in your PR body to link the issue to the PR so it autocloses the issue when merged

are any other material tiles (glass, titanium) affected by this bug?

**Comment by Peliex** [comment]
> are any other material tiles (glass, titanium) affected by this bug?

no, all other tiles properly give 100 units when placed inside the protolathe, at least all the ones that can be printed AT the protolathe, anyway.

File: code/game/objects/items/stacks/tiles/tile_iron.dm [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	icon_state = "tile"
 	inhand_icon_state = "tile"
 	force = 6
-	mats_per_unit = list(/datum/material/iron=SMALL_MATERIAL_AMOUNT*5)
+	mats_per_unit = list(/datum/material/plasma=SHEET_MATERIAL_AMOUNT*0.25)
```
Comment (Sealed101) [inline code/game/objects/items/stacks/tiles/tile_iron.dm:8]: plasma?

Comment (Peliex) [reply code/game/objects/items/stacks/tiles/tile_iron.dm]: copypasted the code from plasma, forgot to change it, fixed in commit d3838d3

**Review by Sealed101 — APPROVED** [review]
LGTM :+1:
seems like the only material tile that hasn't been brought up to match the reworked material cost values was the most used one - the iron tile. _ironic_

**Pull request merged**